libs/core/routing/src/uri/truncation/file-extension.ts

```diff
@@ -2,9 +2,12 @@
  * Truncates the file extension from the end of the URI.
  * Captures the truncated URI in the `uri` named group.
  */
-export const DAFF_TRUNCATE_FILE_EXTENSION_REGEX = /(?<uri>.+)\.(.*)$/;
+export const DAFF_TRUNCATE_FILE_EXTENSION_REGEX = /(.*)(?=\.)/;
 /**
  * Truncates the file extension from the end of the URI.
  */
-export const daffUriTruncateFileExtension = (uri: string): string =>
-  uri.match(DAFF_TRUNCATE_FILE_EXTENSION_REGEX)?.groups.uri || uri;
+export const daffUriTruncateFileExtension = (uri: string): string => {
+  const uriMatch = uri.match(DAFF_TRUNCATE_FILE_EXTENSION_REGEX);
+
+  return uriMatch ? uriMatch[0] : uri;
```

Comment:
FWIW you can do `uri.match(DAFF_TRUNCATE_FILE_EXTENSION_REGEX)?.[0] || uri`

Reply:
makes sense. Changed